File: pfproject_py/pfplayer.py
# !/usr/bin/env python3
# -*- coding:utf-8 -*-
__author__ = 'Zhiquan Wang'
__date__ = '2018/8/3 20:23'


class PixelFightPlayer(object):
    def __init__(self, *, usr_name=None, login_id=None, socket=None):
        self.__usr_name = usr_name
        self.__login_id = login_id
        self.__last_attack_grid = [0, 0]
        self.__socket_info = socket

    @property
    def usr_name(self):
        return self.__usr_name

    @usr_name.setter
    def usr_name(self, _u):
        self.__usr_name = _u

    @property
    def login_id(self):
        return self.__login_id

    @property
    def socket_info(self):
        return self.__socket_info

    @property
    def last_attack_grid(self):
        return self.__last_attack_grid

    @last_attack_grid.setter
    def last_attack_grid(self, _vec):
        self.__last_attack_grid = _vec

File: pfProject_CS/TrialProject/TrialProject/src/PfPlayer.cs
using System;

namespace TrialProject.src{
    public class PixelFightPlayer{
        private string usrName;
        private string loginId;
        private string socketInfo;
        public PixelFightPlayer(String usrName, String loginId, String socket){
            UsrName = usrName;
            LoginId = loginId;
            SocketInfo = socket;
        }
        public PixelFightPlayer() { }
        public string UsrName { get => usrName; set => usrName = value; }
        public string LoginId { get => loginId; set => loginId = value; }
        public string SocketInfo { get => socketInfo; set => socketInfo = value; }

        
    }
}